plugins/Feedback/templates/index.twig

```diff
@@ -8,6 +8,24 @@
 {% block content %}
 
     <div id="feedback-faq" class="admin">
+        <div piwik-content-block content-title="{{ "Feedback_HowCanWeHelp"|translate|e('html_attr') }}" class="feedback-help-resources">
+            <form target="_blank" method="GET" action="https://matomo.org/">
+                <div class="search-matomo-form">
+                    <div>
+                        <span class="icon-search"></span>
+                        <input type="text" placeholder="{{'Feedback_SearchHelpResources'|translate|e('html_attr')}}" name="s"/>
```

Comment:
This isn't using our form widgets like `<div piwik-field uicontrol="text"....>`

Reply:
Might need also generally need to look more like a search field maybe like but with the icon on the left 
![image](https://user-images.githubusercontent.com/273120/64653762-66aa1800-d47b-11e9-94a6-c23f093b2360.png)

BTW we're using MaterializeCSS see http://archives.materializecss.com/0.100.2/shadow.html so we could use classes like `z-depth-1`. If we make the search look like above we would maybe not need to reuse our builtin widgets.

In general be good to improve the look a bit. Could we maybe make it look like this?

![image](https://user-images.githubusercontent.com/273120/64654185-90b00a00-d47c-11e9-811c-f01bb9c1b3ed.png)  (the search icon needs to be vertically centered)